core/src/main/resources/jenkins/diagnostics/URICheckEncodingMonitor/message.jelly

```diff
@@ -0,0 +1,16 @@
+<?jelly escape-by-default='true'?>
+<j:jelly xmlns:j="jelly:core" xmlns:st="jelly:stapler" xmlns:d="jelly:define" xmlns:l="/lib/layout" xmlns:t="/lib/hudson" xmlns:f="/lib/form">
+    <j:if test="${it.checkEnabled}">
+        <script>
```

Comment:
This admin monitor is always active, similar to ReverseProxyMonitor, and only shows a message upon AJAX failure. I just moved the existing code over, it works exactly the same otherwise.